src/components/usage.js

```diff
@@ -0,0 +1,78 @@
+import * as Plot from "npm:@observablehq/plot";
+import { resize } from "npm:@observablehq/stdlib";
+import * as d3 from "npm:d3";
+
+export function weeklyPlot(usageData) {
```

Comment:
i have some ideas for best practices we might want to put together for building plotting components, but ill leave it for later for now. this is fun and i dont want to stand in its way. mostly commenting to remind ourselves we should probably talk about it and then document it at some point so we know were on the same page.